app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -89,18 +90,14 @@ open class FirefoxApplication : LocaleAwareApplication() {
     }
 
     private fun initPush() {
-        var admAvailable = false
-        try {
-            Class.forName("com.amazon.device.messaging.ADM")
-            admAvailable = true
-        } catch (e: ClassNotFoundException) {
-            android.util.Log.i(DEFAULT_LOGTAG, "ADM is not available on this device.")
-        }
-
+        var admAvailable = ADM(applicationContext).isSupported
         // Only install push feature if ADM is available on this device
```

Comment:
nit: -> "Only use push feature..."

I'm not sure what, "install push feature" means.